code/game/machinery/doors/airlock_electronics.dm

```diff
@@ -103,6 +103,15 @@
 			conf_access -= access
 		return TOPIC_REFRESH
 
+/obj/item/stock_parts/circuitboard/airlock_electronics/attackby(obj/item/W, mob/user)
```

Comment:
If this is meant as a passthrough interaction for doors, isn't it just confusing as it conflicts with access checking to open the door? Why not just check access when opening the panel and leave it at that?

Also letting it reset to locked with _any_ ID, including ones that lack access, seems not user friendly.

Reply:
ftr resetting to locked with any ID I did so engineers can finish repairs without grabbing someone with access.